### DISCUSSION TITLE
Self learning In Progress status still not change

### DISCUSSION THREAD

**Discussion by snehadudhankar** [github/india#470] [General]
Future Ready Talent is a virtual internship i am 2 nd batch student. I have completed self learning session but there still showing in progress status .what do I ? can anyone tell me

**Comment by utkarsh-1909** [comment]
the same issue is with me

**Reply by konduyx18** [reply]
Hi I have same problem. Did you solve your problem? If yes, could you please tell me how? Many thanks. Take care 
Yeliz

**Comment by ShivaniPawade** [comment]
same issue
